azure-mgmt-storage/src/main/java/com/microsoft/azure/management/storage/implementation/api/StorageAccountsInner.java

```diff
@@ -122,7 +122,7 @@ public StorageAccountsInner(Retrofit retrofit, StorageManagementClientImpl clien
         final String type = null;
         StorageAccountCheckNameAvailabilityParameters accountName = new StorageAccountCheckNameAvailabilityParameters();
         accountName.setName(name);
-        accountName = null;
+        accountName.setType(type);
```

Comment:
Justification of touching generated code: This is already fixed in AutoRest. Once code is regenerated this will be fixed.